src/Microsoft.ML.Data/Transforms/ValueToKeyMappingEstimator.cs

```diff
@@ -37,7 +37,7 @@ public abstract class ColumnOptionsBase
             public readonly string OutputColumnName;
             public readonly string InputColumnName;
             public readonly SortOrder Sort;
-            public readonly int MaxNumKeys;
+            public readonly int MaxNumberOfKeys;
             public readonly string[] Term;
             public readonly bool TextKeyValues;
```

Comment:
>TextKeyValues [](start = 33, length = 13)

What are you doing?

Reply:
I suggest `KeyValuesAnnotationsAsText`.

---
In reply to: [263042697](https://github.com/dotnet/machinelearning/pull/2862#discussion_r263042697) [](ancestors = 263042697)

Reply:
Maybe `AddKeyValuesAnnotationsAsText`?

---
In reply to: [263223982](https://github.com/dotnet/machinelearning/pull/2862#discussion_r263223982) [](ancestors = 263223982,263042697)